test/externalTests/gnosis.sh

```diff
@@ -31,17 +31,15 @@ function test_fn { npm test; }
 function gnosis_safe_test
 {
     OPTIMIZER_LEVEL=1
-    CONFIG="truffle.js"
+    CONFIG="truffle-config.js"
 
-    truffle_setup "$SOLJSON" https://github.com/solidity-external-tests/safe-contracts.git development_070
+    truffle_setup "$SOLJSON" https://github.com/solidity-external-tests/safe-contracts.git development_070_new
 
-    force_truffle_version ^5.0.42
-    sed -i 's|github:gnosis/mock-contract#sol_0_5_0|github:solidity-external-tests/mock-contract#master_070|g' package.json
+    sed -i 's|github:gnosis/mock-contract#sol_0_5_0|github:solidity-external-tests/mock-contract#master_070_new|g' package.json
     rm -f package-lock.json
     rm -rf node_modules/
 
     run_install "$SOLJSON" install_fn
-    replace_libsolc_call
```

Comment:
Why?

Reply:
Because we're no longer forcing it to use Truffle 5.0.42 and the newer one already uses `solc.compile` rather than `solc.compileStandard` so replacing it is not needed.

Maybe I should also remove the `replace_libsolc_call` function from `common.sh` as it's no longer used...

Reply:
`replace_libsolc_call` removed.